### ISSUE TITLE
Feature Request: Add "Sorting Name" to Sort Field options

### ISSUE THREAD

**Issue by Kahnares** [joyrider3774/Playnite_html_exporter#13]
TL:DR - Please add an option to sort by Playnite's "Sorting Name" field so we can keep series' together.

Let me start by saying that this plugin is awesome. What a really cool way to view information about your games and, if you can host it out on the interwebs, you have a really handy way of showing off your collection and impressing the chicks! (grid view works well for this). Well done! This is one of those things you don't even realize you need...until you do. Well, it is for me anyway...but nobody ever accused me of having a life.

Anyway...

Of all the available options to sort collections, the one I would like the most is missing: Sorting Name. This would be helpful for keeping series of games together. As an easy example, I'll use "Tomb Raider, Rise of the Tomb Raider, Shadow of the Tomb Raider". By default, alphabetical sorting by Name places these in different parts of the library. It would be nice to keep them together as a series. Now, I know that I could probably work around this specific example by sorting using the "Release Date" or "Release Year" and they would likely group together (haven't tested it, by the way).

But...sometimes, developers add a "prequel" game to a series or games within the series may take place at different points in history. In this case, if I wanted to view the series in chronological order, sorting by Release Date/Year won't work. As an example of this, we can look at the "Assassin's Creed" series. Chronologically, this series takes place all over history and the games were not released in any kind of logical chronological order. In Playnite, I can create my own chronogical order using the Sorting Name field. It would be awesome if I could sort by that field in HTML Exporter too.

It could be the option is there and I just don't see it. If so, feel free to give me a kick in the right direction. I also took a peek at the various templates to see if I could edit and/or add the field myself. Sadly, I'm an idiot and I can't read the HTML/CSS/XML/BlackMagic that goes on in those files. Is there any chance of getting that field added to the plugin in a future release?

**Comment by joyrider3774** [comment]
I might have a look at this when i get some time, but not sure its possible, the initial sorting happens on group field and it requires the data to be present. But if i want to sort on it (the groupfield) the i need to the sorting name data to be present but that would mean the original game name won't be displayed but the sortname so not sure i can easily fix this i might be able to use it as a sorting field (after having done an initial sort on groupfield) but i'm not sure that would solve the issue as it would have sorted on name already first then in groupfields but it could work on any other pages than "Name" ones

**Comment by Kahnares** [comment]
Thanks for responding! If I get what you're saying, you're telling me there's no easy solution. I was kinda hoping it was something I overlooked or something that could be fixed with a (small?) change to the voodoo-magic template files. No such luck, I guess. I'll count myself lucky that you might look into it. Maybe it's a sorting feature that other people might be interested in (the OCD folks among us would be eternally grateful). In any case, it's still an awesome plugin and thanks again!
